plugins/org/README-alpha.md

```diff
@@ -0,0 +1,462 @@
+                  singularTitle="My Squad"
+                  pluralTitle="My Squads"
+                  icon={GroupIcon}
+                />
+              </SidebarGroup>
+            </Sidebar>
+          ),
+        };
+      },
+    }),
+  ],
+});
+```
+
+### Entity Cards
+
+The `org` plugin provide some entity cards you can enable to customize the Software Catalog entity page.
+
+> [!IMPORTANT]
+> The order in which cards are listed in the configuration file will determine the order in which they appear in overview cards and tab lists on entity pages.
+
+See a complete cards list below:
+
+#### Entity Group Profile Card
+
+This [entity card](https://github.com/backstage/backstage/blob/master/plugins/catalog-react/api-report-alpha.md) extension allows you to view, edit, or update groups metadata, such as team avatar, name, email, parent, and child groups.
+
+| Kind          | Namespace | Name            | Id                              | Example                                                                                   |
+| ------------- | --------- | --------------- | ------------------------------- | ----------------------------------------------------------------------------------------- |
+| `entity-card` | `org`     | `group-profile` | `entity-card:org/group-profile` | <img alt="Entity Group Profile Card" src="./OrgGroupProfileEntityCard.png" width="200" /> |
+
+##### Disable
+
+This card is disabled by default when you install the `org` plugin, but to ensure the card will always be disabled or enabled regardless of the extension's default definition, add the following configuration:
+
+```yaml
+# app-config.yaml
+# example disabling the org group profile entity card extension
+app:
+  extensions:
+    # this is the extension id and it follows the naming pattern bellow:
+    # <extension-kind>/<plugin-namespace>:<extension-name>
+    # use false as value for disabling the extension and true for enabling
+    - entity-card:org/group-profile: false
+    # or
+    # - entity-card:org/group-profile:
+    #   - config:
+    #       # set 'true' for enabling it again
+    #       disabled: true
+```
+
+##### Config
+
+There is only one configuration available for this entity card extension, which is setting an entity filter that determines when the card should be displayed on the entity page.
+
+Here is an example showing the `group-profile` overview card only for entities of kind group:
+
+```yaml
+# app-config.yaml
+# example setting the extension to only show up for entities with kind "group"
+app:
+  extensions:
+    # this is the extension id and it follows the naming pattern bellow:
+    # <extension-kind>/<plugin-namespace>:<extension-name>
+    - entity-card:org/group-profile:
+        config:
+          # The default value is "kind:group"
+          # For more information about entity cards filters, check out this pull request
+          # https://github.com/backstage/backstage/pull/21480
```

Comment:
Is there documentation for this? Linking to a PR feels kind of wrong here.

Reply:
There are not docs for now unfortunately